### PR TITLE
Move to GHA

### PR DESCRIPTION
**Pull request by orangedeng**
Multiple fixes for GHA ci relocation.

### CODE REVIEW

**Comment by orangedeng** [comment]
Related issue: https://github.com/cnrancher/autok3s/issues/688

File: pkg/settings/setting.go [hunk 1]
```diff
@@ -91,3 +98,8 @@ func GetScriptFromSource(writer io.Writer) error {
 	}
 	return nil
 }
+
+// GetEnvKey will return the given string formatted as a rancher environmental variable.
```
Comment (JacieChao) [inline pkg/settings/setting.go:102]: autok3s environmental variable

**Review by JacieChao — COMMENTED** [review]
LGTM

File: pkg/settings/setting.go (hunk 1 above)
Comment (orangedeng) [reply pkg/settings/setting.go]: Updated

**Review by Jason-ZW — APPROVED** [review]
LGTM

**Review by JacieChao — COMMENTED** [review]
LGTM

**Pull request merged**